ops/ansible/playbooks-ccs/files/client_data_server_dpc_local_certificate.pem

```diff
@@ -1,28 +1,28 @@
-oKbMyiNlnPbiPOWq11qw6apL3skSRTWL0Dnb5JOYQtf1kSd3QwRRYFcstSDOWWC6
-JZjS7WF1gvXkkOmhM9NU+OHt4Odf0OWGLYyn0Axj0TmdSMUtj6yEFywF0qzq0t5X
-UjhdPfdbqomPSs+rBcSeamDTaSHMY/tocW29cD4V4v4lArwnN30Acy5jiUczKwug
-yxMtRiy7XKG7RmcQbSmZkvW3P2iSsmnDps+tTWMfF3L/cm6IQp0v6Cin4qSGGTL6
-NPXwHRX3xFSygJLbW4KUfULP3t4mIiYBLzTfhXhdQKAQPJDnDvSn+0a41cHcrjec
-3TnpnauE0yC0FZfZcS3iOejdn3BsZ7ei/6aD5AOTLyNLDXgoCFafiJrZM7Z/xT5F
-mUJ9bJXUuB9i9bDfAHwcjJERQthGP8xpNh2fi+Zh/QB7c4th1ry4GgrmV/vfQrYC
-chn5JiHTiMiSZfWVUs+YNoBLIxTkJOThEM9RiKrc1I/vrDV8UGs+geSsHxdtAgMB
-AAEwDQYJKoZIhvcNAQELBQADggIBADlB3cXMh9ndz8ahlzIiJ96k3oSQy/BD1XxP
-39RHO3zxpAjLEMCRbNlr76KoCVorjBMVjLw5DLIcAuXuMPTaTbSfuroL/58vMtR2
-gg6Guhm7cnNSpEhldyByPc5sRBlGAbh6SrP//CD52XKfudad01o6LzuABdZKXJeU
-tJx4eBQBj3ch3J8u30KgLDB8KWrP8+lC7h38JqrB8S/MMvA0+0DIFoFLeUrEsxL8
-stSz4jHAf3zvhYc+3YJW1KUbIcfq/413+1mMsIbPrgnKk+Z/T0fFjQvW70+G9Un6
-/UiJck4yTaORTxjvAo8Mvc/4eacFzvldGGqs504L6WIsFO3lx91AG5lbP4YTjyyl
-qIhx4rKOh13woHfZYc/Xnb3bUFOd4LHgXWA48n3vkefxImHpk0+7NBQHMW4/l/sH
-9hP9I+4RzPiEBrGY8Y1IpTUgzSJLs+vWFDpX1tPWeFjR0vMu/nroM5IQQpAzhZOa
-eJ9KdQVLAZHmvmjGJgK0stjo9UmntZgpDZWtIzHbOf2KeHXJCz1EMILZGv16sZAo
-Cy1g7Qp9hs2HeDlY3UjCvyhzWOUAjurnq+RJv2ZMd13AtXmNK70U7XQ5cAT0gPCX
-3OZTTQ5N7Uxeh+zLDYL2TAsBlBt0/0UKt7GALap3GWWYoJHmkwEuJs5eZNxVojGd
-TTdinsBO
+MIIE1TCCAr2gAwIBAgIEBH0XMzANBgkqhkiG9w0BAQwFADAbMRkwFwYDVQQDExBk
+cGMtbG9jYWwtY2xpZW50MB4XDTIwMDIxOTA0MjU0OVoXDTMwMDIxNjA0MjU0OVow
+GzEZMBcGA1UEAxMQZHBjLWxvY2FsLWNsaWVudDCCAiIwDQYJKoZIhvcNAQEBBQAD
+ggIPADCCAgoCggIBAJMGZlIVv3vRMndX6OcV1IK1Ngbu8HUsJFCx9yG8KNp5GT7X
+z7ga18ZwY3Yf5zXrle2KuKpVbSTxy/u/wrD+NEPbwiTnB2RiTbLWCHPO6NuEk3bR
+LAt6p8KZt4uHsl/2L8GnK1d/PqiAisrkvQ1sQy1gUlBQ52jM4iByuA/SbxwQs0fJ
+GfwV3/DfbpYbJpJIW00nKqT6nckpicUBJzyv5TeEE8IkastnABIZiD25gaiF+b+e
+bFTHOSRg0tMIa15/LV54RKd8R+HaIHcJD43yTBBfnpQ07VfqGvzV964tfLqMNEyX
+kSn3zLr1mTG1NVgnyKjQgsRJoOt0xW0Nba7kq4OwS/GAfDnow99ZPARGMfYDe92G
+Z6lqxaYBZrgyvKfHzk6wpI4pPzfGGJC/mCR7q4B1su6eD2sPUAJRnCW4W7dJ+RmW
+S5pdEcjCwHCpB7QUbQzI4FN8CYw6sDzf+LNd/A6B0jsDBT4MGGfm2ELly4KDyeNs
+ZHZAbLV1JZipPpg6QzCwnFt/tPxPhFGidNSK9tKY8hJkqfnm+ReF9F405EUWGswj
+XoO0tnMcJt309qycSrEMgSqpmZYrBDSiX3pvHoV0GdImMHDyeI/cx1poOf+4uXhd
+AjPqqgyRLQP3qDu73232i7DSL/KBZ1eok6KAwjCTZJwU9A8myvfpCr00AqzVAgMB
+AAGjITAfMB0GA1UdDgQWBBRFAa9x6cDeYcX8Gm4wSOhuMay5zDANBgkqhkiG9w0B
+AQwFAAOCAgEAiC/ng3nr5etrDAcKb4BsZiCGE8wLwOi7nlTecppbNrzVN4peVw+q
+WC4JvplWTjWG4xseN3W8wpKOmj2oBlKXBvhV2fs5Hcf75gIqH1dN6TIVUc7Mt5I/
+rwm+PdsxmJ5VrGrerP7KgG4LVPt+eGxLXwoKL1ZsSs3fnk7WPZrVfnbDI9jNombN
+z6Nvq0hq21Iymg0kqlPkF1C6G66NDY/5fjA0ORE9jFDvRfgf1TTWXGrKBgPbctT/
+mWP655raN0FfrXmcGo/Tthko7ftO2+lFXQMnQvVurQ6q2/K1znWOvAv1f7+pNEHU
+OMGCcF1z1koRoLvUTMPnd71JSAFautssai08jokBrqo0EYkHFMxgxOXMU4n+ah4F
+eZi/xP4iKFVvJRpihMaLOx43lhhSz4RBDEoB5xepojhupsV04eY2SIkSZH4pGZej
+Hy3RQipd8v8L2JdEBDATrlIm1jDfc7hwj9oVHU3QxVpAuSGgGAJH/1HUS0UNZ1y9
+ERcnGY36f1aQo+hAbpo6d3XkSncO1NV+8CwLL1V1fImhgxg1nPc8G/yT2nlu/2Ct
+XfmNfZn6i4dJ+XAH1WEhlJh5rGGh/TNKqlEpfORCBFuLG7wCHdCsugPknAhTt7Sl
+40IKfFSSjR0PbpHlwRWXfkVfgTQUFBB4c71xyLKYUAuf0P7LM0a560U=
```

Comment:
openssl x509 -in client_data_server_dpc_local_certificate.pem  -text -noout
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 75306803 (0x47d1733)
    Signature Algorithm: sha384WithRSAEncryption
        Issuer: CN=dpc-local-client
        Validity
            Not Before: Feb 19 04:25:49 2020 GMT
            Not After : Feb 16 04:25:49 2030 GMT
        Subject: CN=dpc-local-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:93:06:66:52:15:bf:7b:d1:32:77:57:e8:e7:15:
                    d4:82:b5:36:06:ee:f0:75:2c:24:50:b1:f7:21:bc:
                    28:da:79:19:3e:d7:cf:b8:1a:d7:c6:70:63:76:1f:
                    e7:35:eb:95:ed:8a:b8:aa:55:6d:24:f1:cb:fb:bf:
                    c2:b0:fe:34:43:db:c2:24:e7:07:64:62:4d:b2:d6: